e2e/test_bootstrap_build_tags.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+# -*- indent-tabs-mode: nil; tab-width: 2; sh-indentation: 2; -*-
+
+# Tests full bootstrap with build tags
+
+SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+source "$SCRIPTDIR/common.sh"
+
+# bootstrap stevedore with 1 change log
+fromager \
+  --log-file="$OUTDIR/bootstrap.log" \
+  --error-log-file="$OUTDIR/fromager-errors.log" \
+  --sdists-repo="$OUTDIR/sdists-repo" \
+  --wheels-repo="$OUTDIR/wheels-repo" \
+  --work-dir="$OUTDIR/work-dir" \
+  --settings-file="$SCRIPTDIR/changelog_settings.yaml" \
+  bootstrap 'stevedore==5.1.0'
+
+EXPECTED_FILES="
+$OUTDIR/wheels-repo/downloads/stevedore-5.1.0-1*.whl
+"
+
+pass=true
+for pattern in $EXPECTED_FILES; do
+  if [ ! -f "${pattern}" ]; then
+    echo "Did not find $pattern" 1>&2
+    pass=false
+  fi
+done
+
+$pass
+
+# bootstrap stevedore again with 1 change log. Should not result in a build
+fromager \
+  --log-file="$OUTDIR/bootstrap.log" \
+  --error-log-file="$OUTDIR/fromager-errors.log" \
+  --sdists-repo="$OUTDIR/sdists-repo" \
+  --wheels-repo="$OUTDIR/wheels-repo" \
+  --work-dir="$OUTDIR/work-dir" \
+  --settings-file="$SCRIPTDIR/bootstrap_settings.yaml" \
+  bootstrap 'stevedore==5.1.0'
+
+# Check for log message that the override is loaded
```

Comment:
This comment isn't accurate. You could probably just delete it, since the error messages make clear what's expected.